src/ol/layers/Layer.js

```diff
@@ -14,7 +14,7 @@ import LayerCommon from '../../common/layers/Layer';
  * @param {boolean} [options.visible=true] If true this layer is the currently visible layer on the map.
  * @param {boolean} [options.isBaseLayer=false] If true this layer is a baseLayer.
  * @param {boolean} [options.isQueryable=true] If true feature information can be queried by the react-spatial LayerService. Default is true.
- * @extends LayerCommon
+ * @extends {Layer}
```

Comment:
What was wrong with LayerCommon?

Reply:
LayerCommon doesn't exists.
Its actual name is Layer, and this way we have the correct inheritance, and we have the correct inherited methods and members in this class.